Cqse.Teamscale.Profiler.Dotnet.sln

```diff
@@ -31,6 +35,22 @@ Global
 		{38D1553A-2E0A-4446-93C2-D3A23836AFD9}.Release|Win32.Build.0 = Release|Any CPU
 		{38D1553A-2E0A-4446-93C2-D3A23836AFD9}.Release|x64.ActiveCfg = Release|Any CPU
 		{38D1553A-2E0A-4446-93C2-D3A23836AFD9}.Release|x64.Build.0 = Release|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Debug|Win32.ActiveCfg = Debug|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Debug|Win32.Build.0 = Debug|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Debug|x64.Build.0 = Debug|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Release|Win32.ActiveCfg = Release|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Release|Win32.Build.0 = Release|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Release|x64.ActiveCfg = Release|Any CPU
+		{A01A7519-E10D-4153-8CD4-678935721DF3}.Release|x64.Build.0 = Release|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Debug|Win32.ActiveCfg = Debug|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Debug|Win32.Build.0 = Debug|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Debug|x64.Build.0 = Debug|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Release|Win32.ActiveCfg = Release|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Release|Win32.Build.0 = Release|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Release|x64.ActiveCfg = Release|Any CPU
+		{1405B69B-D9A6-47F6-9C05-8B839DB3A0E4}.Release|x64.Build.0 = Release|Any CPU
```

Comment:
I've cleaned that up. does it work for you now?

Reply:
LGTM